static/app/views/insights/common/components/modulesOnboarding.tsx

```diff
@@ -376,6 +368,21 @@ const EMPTY_STATE_CONTENT: Record<TitleableModuleNames, EmptyStateContent> = {
       }),
     ],
     imageSrc: assetsPreviewImg,
+    // TODO - this is a lot of manual work, and its duplicated between here and our docs, it would great if there's a single source of truth
+    supportedSdks: [
+      'javascript',
+      'javascript-angular',
+      'javascript-astro',
+      'javascript-ember',
+      'javascript-gatsby',
+      'javascript-nextjs',
+      'javascript-react',
+      'javascript-remix',
+      'javascript-solid',
+      'javascript-svelte',
+      'javascript-sveltekit',
```

Comment:
actually maybe we remove `gatsby`, `ember` and `svelte` because:

- nobodies using `gatsby` or `ember`
- `sveltekit` is what everyone using `svelte` is using anyway

will help save space